io.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,73 @@
+app-id: io.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: cpu-x
+rename-icon: com.github.thetumultuousunicornofdarkness.cpu-x.png
+rename-desktop-file: com.github.thetumultuousunicornofdarkness.cpu-x.desktop
+rename-appdata-file: com.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # OpenGL/Vulkan
+  - --device=dri
+  # Needs use of 'pkexec' to run 'cpu-x-daemon'
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - gtkmm.json
+
+  - name: libcpuid
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DENABLE_DOCS=0
+      - -DBUILD_SHARED_LIBS=0
+    sources:
+      - type: git
+        url: https://github.com/anrieff/libcpuid.git
+        tag: "v0.6.4"
+        commit: 02237e67f5cbff348c63ad5338a70d75909a23b1
+    cleanup:
+      - /bin
+      - /include
+      - '*.a'
+      - /lib/cmake
+      - /lib/pkgconfig
+
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS=0
+      - -DGLFW_BUILD_TESTS=0
+      - -DGLFW_BUILD_DOCS=0
+      - -DGLFW_USE_WAYLAND=1
+    sources:
+      - type: git
+        url: https://github.com/glfw/glfw.git
+        commit: 3fa2360720eeba1964df3c0ecf4b5df8648a8e52
+      - type: patch
+        path: patches/0001-Platform-Prefer-Wayland-over-X11.patch
+    cleanup:
+      - /include
+      - '*.a'
+      - /lib/cmake
+      - /lib/pkgconfig
+
+  - name: cpu-x
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DFLATPAK=1
+    sources:
+      - type: git
+        url: https://github.com/TheTumultuousUnicornOfDarkness/CPU-X.git
+        tag: "v5.0.1"
+        commit: dd629a1fe92a020e1d8fd777c3728e0afb4597dc
+      - type: patch
+        path: patches/0002-fix-launchable-field-in-appdata.patch
```

Comment:
you don't need that patch. `rename-desktop-file` takes care of it.

given that it is still wrong.

Reply:
Ok, patch removed in d24ada277cd6a4caf28b36422404e428e500494b, thanks.